validator/slashing-protection/metrics.go

```diff
@@ -0,0 +1,49 @@
+package slashingprotection
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	// Counts slashable block proposal attempts detected by local slashing protection.
+	localSlashableProposalsTotal = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "slashable_validator_proposals_rejected_total_local",
+			Help: "Counts block proposal attempts rejected by slashing protection.",
+		},
+		[]string{
+			"pubkey",
+		},
```

Comment:
Why by pubkey? This is a total metric and by pubkey is just going to increase cardinality, potentially significantly.